src/Layer/Layer.js

```diff
@@ -2,6 +2,7 @@ import * as THREE from 'three';
 import { STRATEGY_MIN_NETWORK_TRAFFIC } from 'Layer/LayerUpdateStrategy';
 import InfoLayer from 'Layer/InfoLayer';
 import Source from 'Source/Source';
+import { parseSourceData } from 'Provider/DataSourceProvider';
```

Comment:
Do you plan on moving it in this class ?

Reply:
I don't think so because maybe I move the responsibility to the future NodeLayer.

Reply:
Ok good to me